src/models/axis.js

```diff
@@ -165,7 +165,9 @@ nv.models.axis = function() {
                     }
                     if (staggerLabels)
                         xTicks
-                            .attr('transform', function(d,i) { return 'translate(0,' + nv.utils.NaNtoZero((i % 2 == 0 ? '0' : '12')) + ')' });
+                            .attr('transform', function(d,i) {
+                                return 'translate(0,' + (i % 2 == 0 ? '0' : '12') + ')'
```

Comment:
The NaNtoZero is most likely needed here, I had to add it pretty much everywhere in axis.js because it would throw errors in weird situations on multiple graphs.  Several of the defects listed under the novus repo were due to these NaN issues.